Cargo.toml

```
@@ -18,4 +18,5 @@ members = [
     ".",
     "testing/rust-integration",
     "testing/can-messages",
+    "testing/cantools-messages"
```

```suggestion
    "testing/cantools-messages",
```

I like trailing commas because it shortens the diff :)